os-server/src/main/java/app/openschool/user/User.java

```diff
@@ -101,6 +104,12 @@ public class User {
       inverseJoinColumns = @JoinColumn(name = "mentor_id"))
   private Set<User> mentors = new HashSet<>();
 
+  @OneToMany(mappedBy = "user")
+  private List<DiscussionQuestion> discussionQuestionAskPeers;
```

Comment:
why do we need questions and answers mapped to the user? we should not need it here, the opposite should be enough